buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkTextStyles.kt

```diff
@@ -134,23 +147,29 @@ private fun toCompose(source: BpkTextStyles, className: String): TypeSpec {
 
       return FunSpec.constructorBuilder()
         .addModifiers(KModifier.INTERNAL)
-        .addParameter(ParameterSpec
-          .builder(defaultFontFamily, FontFamilyClass)
-          .defaultValue("%T.SansSerif", FontFamilyClass)
-          .build())
-        .callThisConstructor(CodeBlock.builder().addNamed(joinToString(
-          prefix = "\n  ",
-          separator = ",\n  ",
-          postfix = ",\n",
-        ) {
-          "${it.name.toSemanticName()} = ${it.toParameter()}"
-        }, mapOf(
-          "textStyle" to TextStyleClass,
-          "fontWeight" to FontWeightClass,
-          "fontSize" to BpkFontSizeClass,
-          "lineHeight" to BpkLineHeightClass,
-          "letterSpacing" to BpkLetterSpacingClass,
-        )).build())
+        .addParameter(
```

Comment:
this was just a formatting change that happened when i reformatted the file. not sure why detekt didn't pick up changes...